README.md

```diff
@@ -162,24 +40,13 @@ This means that new versions are submitted to a given channel.
 
 Current versions per each channel are represented by the [ModuleTemplate CR](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/technical-reference/api/moduleTemplate-cr.md) submitted to a matching folder in the Kyma Git repository.
 
-Having merged all the changes into the `main` branch in the `keda-manager` repository, the CI/CD jobs will bundle module images and generate ModuleTemplate for you.
+After you merge all the changes into the `main` branch in the `keda-manager` repository, the [CI/CD jobs](/docs/contributor/ci-cd.md) bundle module images and generate the ModuleTemplate for you.
 Submit your ModuleTemplate into the desired channel using a pull request to the Kyma repository.
-A series of governance jobs will start testing if the new candidate version fulfills the criteria described in the [module submission process](https://github.com/kyma-project/community/tree/main/concepts/modularization#module-submission-process).
+A series of governance jobs tests if the new candidate version fulfills the criteria described in the [module submission process](https://github.com/kyma-project/community/tree/main/concepts/modularization#module-submission-process).
 
 
 ## Keda module footprint
```

Comment:
This part is also relevant to the user. Lets move it to the /user folder.

Instead of TBD, lets add the measured values per keda workload:

Keda module consists of four workloads in total.
Two if those ( `keda-manager` and `keda-operator`) are implementing kubernetes operator pattern and consume extra resources only when they detect changes in the resources they watch (i.e Keda CR, ScaledObject CR, etc.. ). They usually are idle and consume as little as few milicores of CPU time and less than 30MB of memory. At the time of active reconciliation the observed CPU time jumps to `5m`.

Similarly to the operators, `keda-admission-webhooks` stays idle most of the time and performs validation operations only when new object ( form `*.keda.sh` api group) are submitted by the user.

The last workload `keda-operator-metrics-apiserver` continuously serves metrics for the kubernetes autoscaling components. Here, the consumption is the highest, but in case for one or two active keda scalers it stays at 5-7 milicores of CPU time.

```

| Name                            | CPU (cores) | Memory (bytes) |
|---------------------------------|-------------|----------------|
| keda-admission-webhooks         | 1m          | 10Mi           |
| keda-manager                    | 3m          | 23Mi           |
| keda-operator                   | 3m          | 26Mi           |
| keda-operator-metrics-apiserver | 5m          | 30Mi           |
```